DiamondLightSource/mx-bluesky
# Remove some warning ignores that are no longer needed

To test:
* Run `./utility_scripts/dls_dev_env.sh` and confirm that the environment is still set up correctly
